### ISSUE TITLE
Tab duplication and loss

### ISSUE THREAD

**Issue by dmatchett** [scripting/drummerSupport#129]
A new issue with Drummer, which has me worried. I noticed my "Scratchpad" tab was missing, so I reopened it from the file menu. It appeared, but in an odd placement off on the right, making a second row for itself, an then it magically renamed itself to "Notes," and yep, suddenly I had two Notes tabs, but no Scratchpad. I can replicate this just by quitting Safari and going through the same steps. The "Empty cache" command from the "Develop" menu, followed by a restart didn't solve the problem either.

**Comment by scripting** [comment]
Yes, I know there are circumstances when one file will overwrite another. I've had it happen to me, but only once and I use Drummer all day every day. I'd like to know more about what you were doing before this happened, if you have any ideas.

The other one is a cosmetic thing, you can fix it by making your window larger, or closing a tab, or living with it. Some people keep a dozen tabs open. I can't manage that much myself.

**Comment by dmatchett** [comment]
Hard to say I was doing anything. I had restarted my laptop yesterday, and this was the first time back to it this morning. AFAIK it was a new instance of everything.

**Comment by dmatchett** [comment]
I experimented a little more with the tabs, so yes, widening the window solved the placement issue. Though I'd seen the second row before, it was the off-on-the-right part that was new to my eye. 

I just retried with a different tab-opening order and got a different result -- if I find a replicable pattern I'll report, of course. Thanks.
